stories/Templates/Authors/Authors.scss

```diff
@@ -8,6 +8,11 @@
     align-items: flex-end;
     .exoandablesearch {
       width: 100%;
+      
+      @include devicebreak(large) {
+        margin-left: -5px;
+      margin-right: -5px;
```

Comment:
@Ashish-srij Fix alignment here. Also why are we using `px` here? Padding and Margins need to be in `rem` unit. Any specific reason why we need negative margins here or can we avoid it?

Reply:
Done